app/views/users/_profile_header.html.erb

```diff
@@ -1,117 +1,69 @@
-            </a>
-          <% end %>
-           <% if @user.instagram_url.present? %>
-            <a href="<%= @user.instagram_url %>" target="_blank" rel="noopener nofollow me">
-              <%= inline_svg_tag("instagram-logo.svg", class: "icon-img", aria: true, title: "Instagram logo") %>
-            </a>
-          <% end %>
-          <% if @user.twitch_url.present? %>
-            <a href="<%= @user.twitch_url %>" target="_blank" rel="noopener nofollow me">
-              <%= inline_svg_tag("twitch-logo.svg", class: "icon-img", aria: true, title: "Twitch logo") %>
-            </a>
-          <% end %>
-          <% if @user.website_url.present? %>
-            <a href="<%= @user.website_url %>" target="_blank" rel="noopener nofollow me">
-              <%= inline_svg_tag("external-link-logo.svg", class: "icon-img", aria: true, title: "external link icon") %>
-            </a>
+        <p class="fs-base m:fs-l color-base-90 mb-4 mx-auto max-w-100 m:max-w-75"><%= sanitize(@user.summary.presence || "404 bio not found") %></p>
+
+        <div class="profile-header__meta">
+          <% if @organization.location.present? %>
+            <span class="profile-header__meta__item">
+              <%= inline_svg_tag("location.svg", aria: true, class: "crayons-icon mr-2 shrink-0", title: "Location") %>
+              <%= sanitize @organization.location %>
+            </span>
           <% end %>
-        <% end %>
-      </p>
-    </div>
+
+          <span class="profile-header__meta__item">
+            <%= inline_svg_tag("cake.svg", aria: true, class: "crayons-icon mr-2 shrink-0", title: "Joined") %>
+            Joined on&nbsp;
+            <%= local_date(@user.created_at) %>
+          </span>
+
+          <span class="profile-header__meta__item -ml-1">
+            <% if @user.twitter_username.present? %>
+              <a href="https://twitter.com/<%= @user.twitter_username %>" target="_blank" rel="noopener me" class="px-1 align-middle inline-block">
+                <%= inline_svg_tag("twitter.svg", class: "crayons-icon", aria: true, title: "Twitter logo") %>
+              </a>
+            <% end %>
+            <% if @user.github_username.present? %>
+              <a href="https://github.com/<%= @user.github_username %>" target="_blank" rel="noopener me" class="px-1 align-middle inline-block">
+                <%= inline_svg_tag("github.svg", class: "crayons-icon", aria: true, title: "GitHub logo") %>
+              </a>
+            <% end %>
+            <% if @user.website_url.present? %>
+              <a href="<%= @user.website_url %>" target="_blank" rel="noopener nofollow me" class="px-1 align-middle inline-block">
+                <%= inline_svg_tag("external.svg", class: "crayons-icon", aria: true, title: "External link icon") %>
+              </a>
+            <% end %>
+          </span>
+        </div>
+      </div>
+
+      <%= render "organizations/metadata" %>
```

Comment:
_Non-blocking nitpick:_

```suggestion
    <% if @organization %>
      <%= render "organizations/metadata" %>
    <% end %>
```

Reply:
despite the name it has (we should rename it), `users/_profile_header` is only rendered for orgs :-| 

```shell
rg "users/profile_header"
app/views/organizations/show.html.erb
6:<%= render "users/profile_header" %>
```

I know...

Reply:
yeap, the current usage of partial is a bit weird but i'm gonna clean it up in a separate PR, where i'll be doing some code cleanups.